app/src/main/res/xml/quran_preferences.xml

```diff
@@ -147,29 +147,16 @@
   </PreferenceCategory>
 
   <PreferenceCategory
```

Comment:
but i don't think this will work because the preference category provides a strange ui on its own - so i think you can just leave this for now.